docs/src/main/sphinx/connector/join-pushdown-enabled-true.fragment

```diff
@@ -10,6 +10,13 @@ improves performance. Note that if no table statistics are available, join
 operation pushdown does not occur to avoid a potential decrease in query
 performance.
 
+.. note::
+
+  The connector performs pushdown where performance may be improved, but in
+  order to preserve correctness an operation may not be pushed down. When
+  pushdown of an operation may result in better performance but risk
+  correctness, the connector prioritizes correctness.
```

Comment:
not limited to join, applies to every kind of pushdown - topN, predicate etc.

Reply:
Can we move this at the beginning of the pushdown section itself? Or a more general page that talks about pushdown?

Reply:
Moved up (in its own fragment) to the parent section. I don't think it makes sense at the _very_ beginning of the section, but certainly after it's been introduced what the connector's pushdown support is. How's that as a compromise?

Reply:
Looks nice. Thanks.